File: 프로그래머스/0/181864. 문자열 바꿔서 찾기/문자열 바꿔서 찾기.java
class Solution {
    public int solution(String myString, String pat) {
        String replaceStr = "";
        for (char word : myString.toCharArray()){
            replaceStr += (word == 'A' ?  "B" : "A");
        }
        return replaceStr.contains(pat) ? 1 : 0;
    }
}

File: 프로그래머스/0/181864. 문자열 바꿔서 찾기/문자열 바꿔서 찾기.cs
using System;

public class Solution {
    public int solution(string myString, string pat) {
        string replaceStr = "";
        for(int i = 0; i < myString.Length; i++){
            replaceStr += (myString[i] == 'A' ? "B" : "A");
        }
        bool contain = replaceStr.Contains(pat);
        return (contain ? 1 : 0);
    }
}